notebooks/mock-notebooks/login/User Login Flow.ipynb

```diff
@@ -0,0 +1,403 @@
+{
```

Comment:
<p>Ahh I see. Forgive me if I'm not up to speed in Python yet. So if verbose=FALSE will they not be able to get error messages?</p><p>Is the idea that we ask them to put in verbose to help troubleshoot when issues happen?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/6333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Additionally some things have been on the mind that I'm not sure if we should tackle now or later is...</p><ul><li>Should we limit the attempts to log in with incorrect passwords within a given timeframe?</li><li>What happens if they forget their password?</li></ul><p> </p><p>I'm curious to know what your thoughts are on the priority of these items and any ideas you have on how to start addressing them.</p>

Reply:
<ol><li>Yes, <code>verbose=True</code>  provides more information/details on the operation being performed, helps a user during troubleshooting.</li><li>For the second comment, great questions there:</li><li class="ql-indent-1">Should we limit attempts... : I think restricting attempts are majorly implemented to prevent malicious attempts to login. It will be a good security measure to do so. On a priority level I feel its a good to have type feature and not very urgent.</li><li class="ql-indent-1">Forget password ... : Yeah, I feel this should be implemented on a higher priority. Because, there is no way where we can reset a user's password (even if I am an Admin). We can have <code>reset_password</code> in both UI and notebook.</li></ol>

Reply:
<p>Yeah cool might create a Feature request story so that we can then loop in Baye and Madhava around the topic. ^_^</p>

Reply:
<p>Story added in shortcut: https://app.shortcut.com/openmined/story/5227/account-settings-reset-password</p>

Reply:
<p>👍</p>